src/main/scala/me/rexim/morganey/Commands.scala

```diff
@@ -6,21 +6,24 @@ object Commands {
 
   final type Command = InterpreterContext => (InterpreterContext, Option[String])
 
-  val commandPattern  = ":([a-zA-Z]+)".r
-  val commandWithArgs = ":([a-zA-Z]+) (.*)".r
+  val commandPattern  = ":([a-zA-Z]*)".r
+  val commandWithArgs = ":([a-zA-Z]*) (.*)".r
```

Comment:
Ok, got that — we _are_ sort of allowing empty commands: it's convenient to think about `:` as a valid command pattern. Okay then.